resen/Resen.py

```diff
@@ -418,10 +420,14 @@ def add_image(self,bucket_name,docker_image):
         
         for x in self.valid_cores:
             if docker_image == x['version']:
-                image = 'docker.io/%s/%s:%s' % (x['org'],x['repo'],x['version'])
+                image = x['version']
+                image_id = x['image_id']
+                pull_image = '%s/%s@%s' % (x['org'],x['repo'],x['repodigest'])
```

Comment:
Notice that `image` is now only the version, and there is a pull_image identifier that has all that is needed to pull the image from docker.io